dmscripts/notify_suppliers_of_new_questions_answers.py

```diff
@@ -0,0 +1,159 @@
+import dmapiclient
+
+from datetime import datetime, date, timedelta
+
+from dmscripts.helpers import env_helpers, logging_helpers
+from dmscripts.helpers.html_helpers import render_html
+from dmscripts.helpers.logging_helpers import logging
+from dmutils.email.dm_mandrill import send_email
+from dmutils.formats import DATETIME_FORMAT
+
+
+EMAIL_SUBJECT = "New question and answer responses on the Digital Marketplace"
```

Comment:
@idavidmcdonald looking at it again I think Kev has a point. 

So, can I ask for one change to this story please? Can we change the email subject line.

From:
New question and answer responses on the Digital Marketplace

To:
New answers to supplier questions on the Digital Marketplace

It feels like maybe we need to clarify what answers, so I've included 'to supplier questions'. It might not be necessary, but I'm including it for clarity.